scripts/utils.py

```diff
@@ -213,6 +213,10 @@ def send_tx(
         "nonce": ledger_api.api.eth.get_transaction_count(crypto.address),
         "chainId": ledger_api.api.eth.chain_id,
     }
+    gas_params = ledger_api.try_get_gas_pricing()
+    if gas_params is not None:
+        raw_tx.update(gas_params)
```

Comment:
I would suggest we add some retries for getting gas price, else we'll end up in the same place as before if the gas pricing failed